static/scss/site-page.scss

```diff
@@ -0,0 +1,17 @@
+.site-page {
+  max-width: $max-width;
+  margin: 0 auto;
+
+  .sidebar {
+    background-color: #eeeeee;
+    border: 1px solid black;
+
+    ul {
+      list-style: none;
+    }
```

Comment:
Small nit: can you introduce a ~15px vertical margin between each immediate child of this `ul`? The top-level menu items are kind of cramped.